client/my-sites/stats/info-panel/index.jsx

```diff
@@ -150,7 +150,7 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/posts/categories-vs-tags/"
 								>
 									<Gridicon icon="info-outline" />
-									{ this.props.translate( 'About Tags & Categories' ) }
+									{ this.props.translate( 'About tags & categories' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.